evalml/pipelines/components/ensemble/stacked_ensemble_regressor.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, input_pipelines=None, final_estimator=None,
                 None and 1 are equivalent. If set to -1, all CPUs are used. For n_jobs below -1, (n_cpus + 1 + n_jobs) are used.
                 Defaults to None.
                 - Note: there could be some multi-process errors thrown for values of `n_jobs != 1`. If this is the case, please use `n_jobs = 1`.
-            random_state (int, np.random.RandomState): Seed for the random number generator
+            random_state (int): Seed for the random number generator
```

Comment:
nit: caps on the S in "Seed".

Reply:
@chukarsten does this mean you prefer `Seed` to be capitalized or not capitalized?

Reply:
@angela97lin I'm going to phone a friend as she just changed a lot of this.  I'm almost 100% sure we're doing caps after the colon.

Reply:
Bahaha yeah, I've been trying to make everything consistently capitalized!